diff --git a/src/FFVship.cpp b/src/FFVship.cpp
index dd59e2c..4f8554f 100644
--- a/src/FFVship.cpp
+++ b/src/FFVship.cpp
@@ -39,8 +39,8 @@ extern "C" {
 #ifndef VSHIP_VERSION_MINOR
 #define VSHIP_VERSION_MINOR 0
 #endif
-#ifndef VSHIP_VERSION_MINOR
-#define VSHIP_VERSION_MINOR 0
+#ifndef VSHIP_VERSION_MINORMINOR
+#define VSHIP_VERSION_MINORMINOR 0
 #endif
 
 using score_tuple_t = std::tuple<double, double, double>;
diff --git a/src/VshipLib.cpp b/src/VshipLib.cpp
index 5425903..2bbfd74 100644
--- a/src/VshipLib.cpp
+++ b/src/VshipLib.cpp
@@ -8,8 +8,8 @@
 #ifndef VSHIP_VERSION_MINOR
 #define VSHIP_VERSION_MINOR 0
 #endif
-#ifndef VSHIP_VERSION_MINOR
-#define VSHIP_VERSION_MINOR 0
+#ifndef VSHIP_VERSION_MINORMINOR
+#define VSHIP_VERSION_MINORMINOR 0
 #endif
 
 #ifdef VULKANBUILD
